server/routes/upload.js

```diff
@@ -29,7 +27,7 @@ module.exports = function({logger, dbClient, batchloadService, authenticationMid
             staged: staged[0].COUNT.value,
             pending: pending[0].COUNT.value,
             errors: errors[0].COUNT.value
-            });
+        });
     }));
 
     router.post('/', asyncMiddleware(async (req, res, next) => {
```

Comment:
The asyncMiddleware is to stop us needing to have try/catch blocks in all the routes. For this application we won't need this as you have added (sensibly I think) try catch below